src/chroot.rs

```diff
@@ -112,9 +115,19 @@ impl Chroot {
         let run = cmd
             .spawn()
             .context(anyhow!("spawn failed. cmd = {cmd:?}"))?;
+
+        // Double Ctrl+C to terminal lium immediately.
```

Comment:
nit: Hit Ctrl-C twice to terminate lium immediately.

Reply:
Ah, it may be a good idea to also add a documentation to explain why this logic is needed (since it is not clear that the SIGINT is sent to the child process as well by the shell even if it is handled in the parent process, adding more details will help the future users / developers to understand the thing clearly ;) )

Reply:
OK.